doc/developer_guide/psyir.rst

```diff
@@ -599,11 +599,10 @@ within PSyclone during their creation (via the ``IntrinsicCall.create``
 function). This attempts to match the Intrinsic and input arguments to
 one of the interfaces for the intrinsic (as some intrinsics have multiple
 possible argument interfaces). If the canonicalisation is successful, PSyclone
-will convert all of the arguments to be named arguments, and reorder arguments
-to match the specification from the Fortran standard. If canonicalisation
-fails, then PSyclone will not create an ``IntrinsicCall`` corresponding to
-the input. This canonicalisation is required to guarantee correct behaviour
-when computing reference_accesses or the return type of an Intrinsic.
+will convert all of the arguments to be named arguments. If canonicalisation
```

Comment:
Please could you reword this whole section (L597-) as I think we've agreed that we're not calling it "canonicalisation" any more?